setup.py

```diff
@@ -811,25 +785,85 @@ def run(self):
 
         def check_linter_exists(linter):
             if shutil.which(linter) is None:
-                msg = "Please install '%s' in your environment. (hint: 'pip install %s')"
+                msg = "Please install '%s' in your environment. (hint: 'python3 -m pip install %s')"
                 warnings.warn(msg % (linter, linter))
                 sys.exit(1)
 
-        c_files_unfiltered = glob.glob("src_c/**/*.[ch]")
-        c_file_disallow = ["_sdl2", "pypm", "SDL_gfx", "sse2neon.h", "src_c/doc/"]
+        c_files_unfiltered = glob.glob("src_c/**/*.[ch]", recursive=True)
+        c_file_disallow = ["_sdl2", "pypm", "SDL_gfx", "sse2neon.h", "src_c/doc/", "_sprite.c"]
         c_files = [x for x in c_files_unfiltered if not any([d for d in c_file_disallow if d in x])]
+
+        # Other files have too many issues for now. setup.py, buildconfig, etc
         python_directories = ["src_py", "test"]
```

Comment:
I wonder if the `examples` directory should be lint checked too? Perhaps that could come in another PR too, it should not block this :)

Reply:
Yeah, I think setup.py, tests, examples and buildconfig can be done in separate PRs.